### ISSUE TITLE
All the readme files have broken links for the widgets

### ISSUE THREAD

**Issue by ArthiSiva** [ToolJet/ToolJet#1354]
### Environment
Documentation

### What is the expected behaviour?
The links to the readme files are broken as there is only alt text available

### What is the current behaviour?
The images should be displayed instead of alt text

I have fixed a few of the issues

**Comment by Navaneeth-pk** [comment]
Can you please mention where we can find these broken links? 
@ArthiSiva

**Comment by TheLearneer** [comment]
@ArthiSiva If you are referring to the `/docs` directory then that is fine. The images get rendered when the site is actually built.

[This readme file](https://github.com/ToolJet/ToolJet/blob/develop/docs/docs/intro.md) has proper rendered image here at [the docs](https://docs.tooljet.io/docs/intro/)
